clients/fides-js/src/components/ConsentModal.tsx

```diff
@@ -1,49 +1,49 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
could also move this to eslintrc: https://github.com/ethyca/fides/blob/feature/privacy-components/clients/admin-ui/.eslintrc.json/#L32